src/resources/lib/modules/connman.py

```diff
@@ -716,7 +731,18 @@ def load_values(self):
+                regValue = regDict.get(regCode)
+            else:
+                regValue = "NONE"
+            self.struct['/net/connman/technology/wifi']['settings']['regdom']['value'] = regValue
```

Comment:
is this writing a value to the connman wireless config via dbus? .. If yes, I'm not sure this is supported (looking in connman online documentation https://git.kernel.org/pub/scm/network/connman/connman.git/tree/doc/config-format.txt).

Reply:
This is just what is displayed in the LE settings windows - it's a current memory load of the settings - so this is loading/setting the display of the default values.